anchore_engine/services/catalog/__init__.py

```diff
@@ -573,8 +584,10 @@ def handle_service_watcher(*args, **kwargs):
                                     ] = taskstate.orphaned_state("service_status")
                                     service_update_record[
                                         "short_description"
-                                    ] = "no heartbeat from service in ({}) seconds".format(
-                                        max_service_orphaned_timer
+                                    ] = generate_error_service_description(
+                                        "no heartbeat from service in ({}) seconds".format(
```

Comment:
Safe/best practice is to use % operator: `"no heartbeat from service in (%s) seconds" % max_service_orphaned_timer`

Reply:
@nightfurys I wasn't aware that using `%` was safer. Could you expand how using that string interpolation would be safer in this case?

I do tend to lean on using `%` when there is nothing to repeat or there is no benefit from using `.format()`. Curious about the best-practice as well